tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier/Program.cs

```diff
@@ -17,23 +25,12 @@ public void Main(string[] args)
                 {
                     var dnxFolder = Environment.GetEnvironmentVariable("DNX_HOME");
```

Comment:
When set `$env:DNX_HOME` may be a semicolon-separated list of paths. When not set, at least `dnvm.ps1` and `dnvm.sh` will fall back to checking `$env:DNX_USER_HOME` and `$env:DNX_GLOBAL_HOME` before assuming a subdirectory of the user's home.